components/ResultView.tsx

```diff
@@ -108,7 +112,7 @@ const ResultView = ({ result }: { result: SearchResult }) => {
   } = result;
 
   const displayName =
-    name && name.raw ? fillHighlights(name.snippet, name.raw) : null;
+    name && name.raw ? fillHighlights(name.snippet, name.raw.bold()) : null;
```

Comment:
If I understand the code correctly there shouldn't be a need to bold `name.raw` since font weight of highlighted text is already set to bold. Could you verify this?